src/daemon/Display.cpp

```diff
@@ -116,16 +116,28 @@ namespace SDDM {
         // Create display server
         switch (m_displayServerType) {
         case X11DisplayServerType:
-            m_terminalId = VirtualTerminal::setUpNewVt();
+            if (seat()->canTTY()) {
+                m_terminalId = VirtualTerminal::setUpNewVt();
+            } else {
+                m_terminalId = -1;
```

Comment:
Can you set the default of `m_terminalId` in the header to `-1` and remove the assignments here?

Reply:
Sure!

Reply:
Done